From cffcf93a8541a2f12cf9882d437670080e2943d5 Mon Sep 17 00:00:00 2001
From: Kun Huang <academicgareth@gmail.com>
Date: Mon, 22 Jul 2013 14:28:10 +0800
Subject: [PATCH] fix r'home/$' in url pattern

r'home/$' will match a container-name like 'test-name' or a path like
'somepath/a-key-name-home'. It should be r'^home/$'

fixes bug #1202079

Change-Id: I94f704990efc4ea6348c5707c774e1141ef1f3c2
(cherry picked from commit e4f5edb9a48400b1c17b590b66395df047b8e965)
---
 horizon/site_urls.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/horizon/site_urls.py b/horizon/site_urls.py
index b25b062..bb62140 100644
--- a/horizon/site_urls.py
+++ b/horizon/site_urls.py
@@ -24,7 +24,7 @@ from django.conf import settings
 
 
 urlpatterns = patterns('horizon.views',
-    url(r'home/$', 'user_home', name='user_home')
+    url(r'^home/$', 'user_home', name='user_home')
 )
 
 # Client-side i18n URLconf.
-- 
1.7.9.5

